components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -1305,6 +1305,10 @@ private ConfigParameters() {
     public static final String X_WSO2_MUTUAL_SSL = "x-wso2-mutual-ssl";
     public static final String X_WSO2_APP_SECURITY = "x-wso2-application-security";
     public static final String X_WSO2_RESPONSE_CACHE = "x-wso2-response-cache";
+    public static final String X_WSO2_DISABLE_SECURITY = "x-wso2-disable-security";
```

Comment:
isn't these key names are already defined somewhere?

Reply:
No. These keys are not defined in anywhere

Reply:
Isn't these are added by APIM?

Reply:
Do we even add x-wso2-disable-security in API Manager? If not, why do we even try to remove it?

Reply:
Got it. You are trying to remove all mgw supported extensions in APIM I believe, no matter APIM supports them or not. Please add this to the PR description and add comments in the code to avoid confusion.

Reply:
No. When someone importing a microgateway compatible swagger file with vendor extensions, this issue is reproducible. Better we remove them if they exist. @praminda is it? 

The issue has been reported at wso2/product-apim#5162

Reply:
@AmaliMatharaarachchi yes Correct. +1

Reply:
Is there a use case to import MG generated swagger to APIM? I think APIM doesn't have to manage extension not added by APIM.